src/backend/distributed/metadata/node_metadata.c

```diff
@@ -142,11 +150,11 @@ DefaultNodeMetadata()
 
 
 /*
- * master_add_node function adds a new node to the cluster and returns its id. It also
+ * citus_add_node function adds a new node to the cluster and returns its id. It also
  * replicates all reference tables to the new node.
  */
 Datum
-master_add_node(PG_FUNCTION_ARGS)
+citus_add_node(PG_FUNCTION_ARGS)
```

Comment:
function changes like this will possibly break tools like citus_dev, do we have a preparation for that?

Reply:
my bad, I thought we were replacing these functions.